src/test/java/be/seeseemelk/mockbukkit/entity/EntityEquipmentMockTest.java

```diff
@@ -125,4 +125,70 @@ void testBoots()
 		assertEquals(item, equipment.getItem(EquipmentSlot.FEET));
 	}
 
+	@Test
+	void testMainHandDropChance()
+	{
+		ArmorStand armorStand = new ArmorStandMock(server, UUID.randomUUID());
+		EntityEquipment equipment = armorStand.getEquipment();
+
+		equipment.setItemInMainHandDropChance(0.75f);
+
+		assertEquals(0.75f, equipment.getItemInMainHandDropChance());
+	}
+
+	@Test
+	void testOffHandDropChance()
+	{
+		ArmorStand armorStand = new ArmorStandMock(server, UUID.randomUUID());
+		EntityEquipment equipment = armorStand.getEquipment();
+
+		equipment.setItemInOffHandDropChance(0.75f);
+
+		assertEquals(0.75f, equipment.getItemInOffHandDropChance());
+	}
+
+	@Test
+	void testHelmetDropChance()
+	{
+		ArmorStand armorStand = new ArmorStandMock(server, UUID.randomUUID());
+		EntityEquipment equipment = armorStand.getEquipment();
+
+		equipment.setHelmetDropChance(0.75f);
+
+		assertEquals(0.75f, equipment.getHelmetDropChance());
+	}
+
+	@Test
+	void testChestplateDropChance()
+	{
+		ArmorStand armorStand = new ArmorStandMock(server, UUID.randomUUID());
+		EntityEquipment equipment = armorStand.getEquipment();
+
+		equipment.setChestplateDropChance(0.75f);
+
+		assertEquals(0.75f, equipment.getChestplateDropChance());
+	}
+
+	@Test
+	void testLeggingsDropChance()
+	{
+		ArmorStand armorStand = new ArmorStandMock(server, UUID.randomUUID());
+		EntityEquipment equipment = armorStand.getEquipment();
+
+		equipment.setLeggingsDropChance(0.75f);
+
+		assertEquals(0.75f, equipment.getLeggingsDropChance());
+	}
+
+	@Test
+	void testBootsDropChance()
+	{
+		ArmorStand armorStand = new ArmorStandMock(server, UUID.randomUUID());
+		EntityEquipment equipment = armorStand.getEquipment();
+
+		equipment.setBootsDropChance(0.75f);
+
+		assertEquals(0.75f, equipment.getBootsDropChance());
+	}
```

Comment:
I am not quite sure if ArmorStands are a good subject to run these tests on as I am not entirely sure whether they even support drop chances in the first place.

Also do keep in mind that EntityEquipment does not work on insentient entities:
https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/browse/src/main/java/org/bukkit/craftbukkit/inventory/CraftEntityEquipment.java#250

I don't quite remember whether ArmorStands do count as such, so better check that out and adjust the code accordingly if required. Sadly, Bukkit has no Insentient mob interface though, so that exclusion may need to be manual.
Regarding the unit tests, a `ZombieMock` may be a better test subject.

Reply:
Yup armor stands aren't insentient. Looking at the sources it looks like EntityInsentient is about equivalent to the Mob interface in function so I've made it check for that.